inject_test.go

```diff
@@ -204,33 +203,6 @@ func TestInject(t *testing.T) {
 		assert.True(t, gave3 == out.T3, "T3 must match")
 	})
 
-	t.Run("OverwritesExisting", func(t *testing.T) {
```

Comment:
Note that this test no longer applies. It was trying to overwrite a value of an already-injected struct. 

This no longer applies since Start no longer modifies the graph, it's fully resolved.